### ISSUE TITLE
After applying reflow label present for different section of chart is not visible :A11y_.NET maui_User can get all the insights of Dashboard_Reflow

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30802]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG1.4.10; #BM_.NETMaui_Win32_JUL2025; #DesktopApp; #Win32; #FTP; #A11ySev2; #.NET MAUI SDKs; #A11yWCAG2.1, #Closed;

## Environment Details:
Application: Developer Balance (.NET Maui)
Windows 11
OS Build: 26100.2161
Version 24H2

### Prerequisites to apply reflow:
Set the width of page to 320

## Repro Steps:

1. Install and open the Developer Balance app
3. Apply  reflow
5. Observe the issue 

## Actual Result:
After applying reflow label present for different section of chart is not visible.

## Expected Result:
After applying reflow label present for different section of chart should be visible.

## User Impact:
Low vision user who uses application in very small port will unable to understand the information of chat.

## Attachment

<img width="1279" height="725" alt="Image" src="https://github.com/user-attachments/assets/08d170bd-b839-4160-8140-4db1582ac738" />

**Comment by ghost** [comment]
This issue has been verified in Visual Studio 17.14.0 Preview 4.0 with MAUI versions (9.0.0 & 9.0.90). It can be reproduced on Windows platform.

**Comment by kapilvaishna** [comment]
@deepika2509  for you information i have changed the description for this bug.

**Comment by kapilvaishna** [comment]
@PureWeen  For this one you mention as below on mail: .NET MAUI SDKs Compliance Status

<img width="336" height="52" alt="Image" src="https://github.com/user-attachments/assets/e954ca2e-45db-428a-917a-b1faa791df4c" />

did  you mean its external to WinUI, if yes we will follow external bug process.

Could you please confirm Win UI is 1st party or 3rd Party. If 1st party could you please share the service tree ID of WinUI team and if 3rd party please share sample application to follow external bug process.

**Comment by PureWeen** [comment]
> [@PureWeen](https://github.com/PureWeen) For this one you mention as below on mail: .NET MAUI SDKs Compliance Status
> 
> <img alt="Image" width="336" height="52" src="https://private-user-images.githubusercontent.com/124254981/502459744-e954ca2e-45db-428a-917a-b1faa791df4c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjExNTQyOTQsIm5iZiI6MTc2MTE1Mzk5NCwicGF0aCI6Ii8xMjQyNTQ5ODEvNTAyNDU5NzQ0LWU5NTRjYTJlLTQ1ZGItNDI4YS05MTdhLWIxZmFhNzkxZGY0Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyMlQxNzI2MzRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Nzg0ODRlYWQwY2RiMjZmN2JmMGUzMTdiNzlmYmRhZGU2MmUyYzRkMGFmN2YxMmM4ZTdhNjI3YzY3YWM1ZTIyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.TzHyNGfSHLAfuYt0n8PVJDGXdDkhlPYGjkMzut0eJYQ">
> did you mean its external to WinUI, if yes we will follow external bug process.
> 
> Could you please confirm Win UI is 1st party or 3rd Party. If 1st party could you please share the service tree ID of WinUI team and if 3rd party please share sample application to follow external bug process.

@kapilvaishna 

I was mainly indicating here that this is an upstream bug with Syncfusion

apologies, my language wasn't super clear in the email

**Comment by kapilvaishna** [comment]
Issue is not repro with Magnifier so closing the bug

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/07c5642d-4eb0-4125-95dc-809f96ff7674" />
